src/main/java/internetshop/controller/CreateBucketController.java

```diff
@@ -15,14 +15,11 @@
     @Inject
```

Comment:
let's rename this class to BucketController or smth else

Reply:
Done